generators/chipyard/src/main/scala/ChipTop.scala

```diff
@@ -31,11 +31,15 @@ abstract class BaseChipTop()(implicit val p: Parameters) extends RawModule with
   val systemReset = Wire(Input(Bool()))
 
   // The system module specified by BuildSystem
-  val system = withClockAndReset(systemClock, systemReset) { p(BuildSystem)(p) }
+  val lSystem = p(BuildSystem)(p)
```

Comment:
```suggestion
  val lSystem = p(BuildSystem)(p).suggestName("system")
```

Reply:
I've applied this to a local change i'll push tomorrow.